src/components/VetList.tsx

```diff
@@ -37,9 +37,9 @@ export const VetList = () => {
   }, []);
 
   const [page, setPage] = useState(0);
-  const [rowsPerPage, setRowsPerPage] = useState(8);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
```

Comment:
question. I've noticed that this line changed as well, https://github.com/firasrg/spring-petclinic-reactjs/pull/8/files#diff-60f8fe10beb84f4dee5a8f257de1993c9e046d84b155b6d665b07281e4b983b2R26

If that's going to be consistent change, either you can specify in config somewhere; or try to provide this number into component props

Reply:
thanks! for now I have updated and extracted to constants in the same files. So we don't have magic numbers anymore, but not yet using env configuration. As it is an application for beginners, it might be too complicated to have indirections in config files (I think we can discuss it and decide later).